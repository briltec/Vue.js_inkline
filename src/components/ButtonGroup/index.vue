<template src="./template.html" />
<script src="./script.js" />
<style src="../Button/style.styl" lang="stylus" scoped />
<style src="./style.styl" lang="stylus" scoped />
