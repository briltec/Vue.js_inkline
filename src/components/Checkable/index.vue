<script src="./script.js" />
<!--<style src="./style.styl" lang="stylus" />-->
