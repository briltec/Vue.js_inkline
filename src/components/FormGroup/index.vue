<template src="template.html" />
<script src="script.js" />
<style src="../Input/style.styl" lang="stylus" scoped />
<style src="style.styl" lang="stylus" scoped />
