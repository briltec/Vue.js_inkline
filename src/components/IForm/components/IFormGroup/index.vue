<template src="./template.html" />
<style src="./style.scss" lang="scss" />
<script src="./script.js" />
