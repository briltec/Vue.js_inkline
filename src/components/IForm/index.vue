<template src="./template.html" />
<script src="./script.ts" lang="ts" />
<style src="../IInput/style.scss" lang="scss" scoped />
<style src="./style.scss" lang="scss" scoped />
