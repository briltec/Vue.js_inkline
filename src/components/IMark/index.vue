<template src="./template.html" />
<script src="./script.ts" lang="ts" />
