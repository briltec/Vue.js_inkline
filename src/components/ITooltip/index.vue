<template src="./template.html" />
<script src="./script.ts" lang="ts" />
<style src="./style.scss" lang="scss" />
