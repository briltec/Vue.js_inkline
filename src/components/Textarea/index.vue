<template src="./template.html" />
<style src="../Input/style.styl" lang="stylus" />
<script src="./script.js" />
