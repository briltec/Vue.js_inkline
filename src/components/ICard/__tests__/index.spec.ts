import { shallowMount } from '@vue/test-utils';
import { ICard } from '@inkline/inkline/components';

describe('Components', () => {
    describe('ICard', () => {
        const props = {
            color: 'light',
            size: 'md'
        };

        it('should be named correctly', () => {
            expect(ICard.name).toEqual('ICard');
        });

        it('should render correctly', () => {
            const wrapper = shallowMount(ICard, { props });
            expect(wrapper.html()).toMatchSnapshot();
        });

        describe('computed', () => {
            describe('classes', () => {
                it('should return classes object', () => {
                    const wrapper: any = shallowMount(ICard, { props });

                    expect(wrapper.vm.classes).toEqual({
                        [`-${wrapper.vm.color}`]: true,
                        [`-${wrapper.vm.size}`]: true
                    });
                });
            });
        });
    });
});
