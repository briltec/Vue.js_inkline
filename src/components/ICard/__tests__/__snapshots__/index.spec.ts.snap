// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components ICard should render correctly 1`] = `
<div class="card -light -md">
  <!--v-if-->
  <!--v-if-->
  <!--v-if-->
</div>
`;
